Exercises_lvl_1.py

```diff
@@ -0,0 +1,23 @@
+import random
+
+number = random.randint(1,51)
+print(f'test: {number}')
+global try_num
+try_num = 0
+rerun = 1
+print("Guess the number!")
+print("Type 'stop' to exit")
+while rerun:
+    try:
+        quess = input("Put the number: ")
+    except:
+        print("Blee")
+        continue
+    try_num = try_num + 1
+    if int(quess) == number:
```

Comment:
Lepiej przetworzyć guess na int tylko raz podczas wprowadzenia tej wartości niż robić to 2 razy tutaj i warunek niżej